README.md

```diff
@@ -1,4 +1,4 @@
-# 4th-year-project
+# snapAll
 
 This repository contains development and deployment files of 4th year Major Project
 
```

Comment:
Please add a small description of the project, means, what this project is all about.